vsintegration/tests/unittests/HelpContextServiceTests.fs

```diff
@@ -1,50 +1,75 @@
 ﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-namespace Tests.LanguageService.F1Keyword
+namespace Microsoft.VisualStudio.FSharp.Editor.Tests.Roslyn
 
 open System
+open System.Threading
+
 open NUnit.Framework
-open Salsa.Salsa
-open Salsa.VsOpsUtils
-open UnitTests.TestLib.Salsa
+
+open Microsoft.CodeAnalysis.Classification
+open Microsoft.CodeAnalysis.Editor
+open Microsoft.CodeAnalysis.Text
+open Microsoft.CodeAnalysis
+open Microsoft.FSharp.Compiler.SourceCodeServices
+open Microsoft.VisualStudio.FSharp.Editor
+open Microsoft.VisualStudio.FSharp.LanguageService
 open UnitTests.TestLib.Utils
-open UnitTests.TestLib.LanguageService
-open UnitTests.TestLib.ProjectSystem
 
-[<TestFixture>] 
-type UsingMSBuild() =
-    inherit LanguageServiceBaseTests()
+[<TestFixture>]
+type HelpContextServiceTests() =
 
-    member private this.TestF1Keywords(expectedKeywords, testLines, ?addtlRefAssy : list<string>) =
-        let lines, poslist = // poslist is in _cursor_ postions. Cursor postions are 1-based.
-            let rec extractPos (col:int) row (s:string) poslist =
-                    let next = s.IndexOf("$", col)
-                    if next < 0 then
-                        s, poslist
-                    else                     
-                        let s = s.Remove(next, 1)
-                        extractPos next row s ((row,next+1)::poslist)
-            let _, l, p =
-                List.fold (fun (row,lines,poslist) s -> 
-                                let (s', poslist') = extractPos 0 row s poslist
-                                (row+1, (s'::lines),poslist')) (1,[],[]) testLines 
-            List.rev l , List.rev p
+    let fileName = "C:\\test.fs"
```

Comment:
That was my understanding too. This exists in other tests [here](https://github.com/Microsoft/visualfsharp/blob/99710407cee56843ed701b9643ac5e2edc8f1c2a/vsintegration/tests/unittests/BreakpointResolutionService.fs#L22), [here](https://github.com/Microsoft/visualfsharp/blob/f67db1f85f9d30c62ff388bc036c6d9064d87766/vsintegration/tests/unittests/LanguageDebugInfoServiceTests.fs#L22).
`ParseAndCheckFileInProject` gets the source code as a string.